src/Identity/Controllers/SsoController.cs

```diff
@@ -86,9 +95,11 @@ public async Task<IActionResult> PreValidate(string domainHint)
         public async Task<IActionResult> Login(string returnUrl)
         {
             var context = await _interaction.GetAuthorizationContextAsync(returnUrl);
+            var query = Request.Query;
```

Comment:
Doesn't look like the `query` parameter is being used, this line should be deleted.